blender-mesh/src/combine_indices/mod.rs

```diff
@@ -73,6 +73,9 @@ impl BlenderMesh {
         let mut expanded_normals = vec![];
         expanded_normals.resize((largest_vert_id + 1) * 3, EASILY_RECOGNIZABLE_NUMBER);
 
+        let mut expanded_material_index = vec![];
+        expanded_material_index.resize( largest_vert_id +1, 0u16);
```

Comment:
Mind using the `EASILY_RECOGNIZABLE_NUMBER`. Meant to help more easily see that data is wrong. Which will help in the future this crate gets heavily refactored..